k8s/client.go

```diff
@@ -1084,6 +1085,22 @@ func (c *Client) GetHelmState(ctx context.Context, namespace string, secretName
 	}, nil
 }
 
+// GetHelmValues is the function for cilium cli sysdump to collect the helm values from the release directly
+func (c *Client) GetHelmValues(_ context.Context, releaseName string) (string, error) {
+	helmGetValsClient := action.NewGetValues(c.HelmActionConfig)
```

Comment:
If I would like to honor `--cilium-namespace` or` --tetragon-namespace`flag than `-n`, is it better to init a k8s client just for get helm values so it will not use `-n`. the default k8s client was init when the cilium-cli command start(https://github.com/liyihuang/cilium-cli/blob/main/internal/cli/cmd/cmd.go#L43), I dont have the chance to override it. I'm also afraid that if I change the default k8s client's NS, it will cause other issues since there will more some goroutine will use the default k8s client to perform the sysdump

Reply:
If we really want to support `--cilium-namespace` for this, then I would take the following approach. Initialize your own HelmActionConfig similar to [how it's done in the k8s package](https://github.com/liyihuang/cilium-cli/blob/ccc2fcb396d4706df68cb39b27d7882f0c193108/k8s/client.go#L127-L131), but instead within the `sysdump` codepath. You can use k8s exported RESTClientGetter for this, but set a different namespace.

That being said, can you explain to me why we might want to support these flags? I'm sure that I followed in the Slack thread. I thought we determined that we might be able to remove these flags?

Reply:
I thought we can remove this `--cilium-namespace` but I noticed it has another flag `tetragon-namespace` so it looks like we need to support this kind of flag anyway.

for the implementation, I noticed our `RESTClientGetter` is an interface, it looks like there are more effort to use it in sysdump. Also I have some concern, if I reuse this `RESTClientGetter` and to set a different namespace, it's a default k8s client behaviour change. there are multiple goroutine is using this default k8s in sysdump. Won't it cause problems?